src/m365/planner/commands/bucket/bucket-set.spec.ts

```diff
@@ -383,6 +349,36 @@ describe(commands.BUCKET_SET, () => {
     }));
   });
 
+  it('Correctly updates bucket by name with rosterId', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/planner/rosters/${validRosterId}/plans`) {
+        return planResponse;
+      }
+      if (opts.url === `https://graph.microsoft.com/v1.0/planner/plans/${validPlanId}/buckets`) {
+        return singleBucketByNameResponse;
+      }
+
+      throw 'Invalid Request';
+    });
+    sinon.stub(request, 'patch').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/planner/buckets/${validBucketId}`) {
+        return '';
+      }
+
+      throw 'Invalid Request';
+    });
+
+    await assert.doesNotReject(command.action(logger, {
+      options: {
+        name: validBucketName,
+        rosterId: validRosterId,
+        ownerGroupName: validOwnerGroupName,
+        newName: 'New bucket name',
+        orderHint: validOrderHint
+      }
+    }));
```

Comment:
Currently, we are not testing anything. Let's check if the patch request was actually fired.